File: java_trans/src/main/java/com/system/schedule/dto/MyInformation/MyInformationRes.java
package com.system.schedule.dto.myinformation;

import lombok.Getter;
import lombok.Setter;
import java.time.LocalDateTime;

@Getter
@Setter
public class MyInformationRes {
    private String infoId;
    private String types;
    private String title;
    private String content;
    private LocalDateTime publishTime = LocalDateTime.now();
}

File: java_trans/src/main/java/com/system/schedule/dto/MyInformation/MyInformationRes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model.Dto.MyInformation
{
    public class MyInformationRes
    {
        public string InfoId { get; set; }
        public string Types { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime PublishTime { get; set; } = DateTime.Now;
    }
}
